allennlp/modules/similarity_functions/linear.py

```diff
@@ -53,6 +55,12 @@ def __init__(self,
         self._weight_vector = Parameter(torch.Tensor(combined_dim))
         self._bias = Parameter(torch.Tensor(1))
         self._activation = activation
+        self.reset_parameters()
+
+    def reset_parameters(self):
+        std = 6.0 / math.sqrt(self._weight_vector.size(0))
```

Comment:
For "xavier_uniform" (aka glorot_uniform) the 6.0 should be inside the sqrt as well. Also, technically, the denominator is `fan_in + fan_out`, where for a vector I guess `fan_out = 1` so doesn't really matter (I'm unclear on how well this init works for vectors).